frontend/src/logout.ts

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const app = express();
+
+app.get("/logout", (req, res) =>{

```

Comment:
This creates a new route, which I don't think we want. I think having a function that just calls `removeCookie("access_token")` would suffice (need to include `removeCookie`). That way, we can clear this from anywhere without having to navigate to a url.